### ISSUE TITLE
Preact Compatibility

### ISSUE THREAD

**Issue by jaredpalmer** [RafalFilipek/jsxfromhtml#9]
Saw that this has a Preact github tag? If it's compatible then we should make that more explicit in docs.

**Comment by RafalFilipek** [comment]
True. Preact and Inferno are mentioned in first paragraph in README. I will rewrite this part. Also README In demo directory should have some info how to run it with Preact and Inferno.
